### DISCUSSION TITLE
logCDF for gamma distribution

### DISCUSSION THREAD

**Discussion by ghalimi** [stdlib-js/stdlib#388] [Q&A] [answered]
Is there a reason why the [gamma distribution](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/gamma) implements `logPDF` but not `logCDF`?

**Comment by kgryte** [comment, marked as answer]
We often look for specialized implementations for better accuracy. When such a specialized implementation is hard to come by, we typically implement a naive algorithm. This has now been [implemented](https://github.com/stdlib-js/stdlib/tree/6097484dcb6619a58fbfbcd0d92be83e50801a52/lib/node_modules/%40stdlib/stats/base/dists/gamma/logcdf).
